packages/css/src/components/dialog/dialog.scss

```diff
@@ -21,18 +27,14 @@
 .ams-dialog__form {
   display: grid;
   gap: var(--ams-dialog-form-gap);
-  grid-template-rows: auto 1fr auto;
-  max-block-size: var(--ams-dialog-form-max-block-size);
```

Comment:
This broke the scrolling inside the Dialog (while keeping the action buttons in view).